src/convert.jl

```diff
@@ -1026,3 +1026,57 @@ function convert(::Type{HParallelotope}, Z::AbstractZonotope{N}) where {N}
     end
     return HParallelotope(D, c)
 end
+
+"""
+    convert(::Type{Zonotope}, cp::CartesianProduct{N, <:AbstractZonotope{N}
+            , <:AbstractZonotope{N}}) where {N}
+
+Converts a cartesian product of two zonotopes into a zonotope.
+
+### Input
+
+- `Zonotope` -- type used for dispatch
+- `cp`       -- cartesian product of two zonotopes
+
+### Output
+
+A zonotope.
+
+### Notes
+
+This function requires the use of `SparseArrays`.
+"""
+function convert(::Type{Zonotope}, cp::CartesianProduct{N, <:AbstractZonotope{N}
+                 , <:AbstractZonotope{N}}) where {N}
+    Z1, Z2 = cp.X, cp.Y
+    c = sparse(vcat(center(Z1), center(Z2)))
```

Comment:
Why is the center vector sparse? I think it is completely dense if the original centers did not contain any zeros, right? Using a sparse representation for a dense vector is inefficient.

The same comment applies to the `CPA` implementation.

Reply:
this was to match the generators matrix which is sparse by using blockdiag.

Reply:
Should I keep them sparse?

Reply:
if the component zonotopes of the cp are sparse => definitely yes

otherwise, well it depends.. we cnan let `vcat(center(Z1), center(Z2))` such that if `Z1` and `Z2` are sparse => the result is sparse, and dens if they are dense.

Reply:
Agreed.